### PR TITLE
[photoprism] Add MariaDB support to Photoprism

### PR DESCRIPTION
**Pull request by djjudas21**
**Description of the change**

Add support for MariaDB database backend to Photoprism for increased app performance. The style of the change was copied from the Webtrees chart, which uses the Bitnami MariaDB chart. All options are passed to Photoprism by env vars.

The default behaviour has been retained (i.e. SQLite remains the default backend).

https://docs.photoprism.app/getting-started/config-options/#database-connection

**Benefits**

Photoprism will be able to use MariaDB as well as SQLite, which will offer better performance for large photo libraries.

**Possible drawbacks**

None

**Additional information**

I'm not sure if the `README.md` is partially generated by CI, but I documented the new params in `values.yaml` and also in `README.md` in case it wasn't automagically re-generated. Happy to rework this if necessary.

**Checklist** 
- [X] Title of the PR starts with chart name (e.g. `[home-assistant]`)
- [X] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [X] Chart `artifacthub.io/changes` changelog annotation has been updated in `Chart.yaml`. See [Artifact Hub documentation](https://artifacthub.io/docs/topics/annotations/helm/#supported-annotations) for more info.
- [X] Variables have been documented in the `values.yaml` file.

### CODE REVIEW

**Comment by bjw-s** [comment]
Thanks a lot for creating this PR!! Sorry it took me a few days to get to it 😅 The changes look good, I'll merge once the CI passes 👍 

> I'm not sure if the README.md is partially generated by CI, but I documented the new params in values.yaml and also in README.md in case it wasn't automagically re-generated. Happy to rework this if necessary.

The `README.md` file will be automatically generated when the PR is merged into the `main` branch. Any manual changes to the `README.md` file will be overridden. We'll try to add that somewhere more explicitly in the PR template.

**Review by bjw-s — APPROVED** [review]

**Comment by djjudas21** [comment]
Thanks @bjw-s. I also noticed the `appVersion` tag needs to be bumped. Do you want a separate PR for that, or can I tag it on here? I've been running `20220121` for a while by overriding it in `values.yaml` so I'm confident it's stable.

**Comment by bjw-s** [comment]
> Thanks @bjw-s. I also noticed the `appVersion` tag needs to be bumped. Do you want a separate PR for that, or can I tag it on here? I've been running `20220121` for a while by overriding it in `values.yaml` so I'm confident it's stable.

Feel free to add it in this one :) I'll hold off on merging the PR and will re-approve once you've pushed the change

**Comment by bjw-s** [comment]
Could you then maybe also update the values.yaml to something similar to this?
https://github.com/k8s-at-home/charts/blob/master/charts/stable/paperless/values.yaml#L13L15
It makes it so that we only have to update the appVersion (by default) but it still allows users to override it if they want.

File: charts/stable/photoprism/Chart.yaml
Comment (bjw-s) [inline charts/stable/photoprism/Chart.yaml:31]: Don't forget to add an entry for the updated application version 🙂

Comment (djjudas21) [reply charts/stable/photoprism/Chart.yaml]: Done

File: charts/stable/photoprism/Chart.yaml
Comment (bjw-s) [inline charts/stable/photoprism/Chart.yaml:31]:
```suggestion
      description: Bumped application version to `20220121`.
    - kind: added
      description: Added MariaDB database support.
```

**Review by bjw-s — CHANGES_REQUESTED** [review]

**Comment by djjudas21** [comment]
Cool, I didn't realise you could have multiple changelog messages. Looks good to me

**Review by bjw-s — APPROVED** [review]

**Comment by bjw-s** [comment]
@all-contributors please add @djjudas21 for code

**Pull request merged**
